From 5cc1313321c3e8cad9726968793c882200e0f850 Mon Sep 17 00:00:00 2001
From: Florian Albrechtskirchinger <falbrechtskirchinger@gmail.com>
Date: Wed, 6 Jul 2016 06:36:50 +0200
Subject: [PATCH] systemd: install missing drop-in configs

In host-only mode, install missing drop-in configuration files for
/etc/systemd/{journal.conf,system.conf}.
---
 modules.d/00systemd/module-setup.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index e4260c77..51904c56 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -162,7 +162,9 @@ install() {
     if [[ $hostonly ]]; then
         inst_multiple -H -o \
             /etc/systemd/journald.conf \
+            /etc/systemd/journald.conf.d/*.conf \
             /etc/systemd/system.conf \
+            /etc/systemd/system.conf.d/*.conf \
             /etc/hostname \
             /etc/machine-id \
             /etc/machine-info \
